Formula/traildb.rb

```diff
@@ -20,19 +20,27 @@ class Traildb < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "python@3.10" => :build
   depends_on "libarchive"
 
   resource "judy" do
     url "https://downloads.sourceforge.net/project/judy/judy/Judy-1.0.5/Judy-1.0.5.tar.gz"
     sha256 "d2704089f85fdb6f2cd7e77be21170ced4b4375c03ef1ad4cf1075bd414a63eb"
   end
 
+  # Update waf script for Python 3
+  patch do
+    url "https://github.com/traildb/traildb/commit/12ccff42e73219e2732ffd7f4ee42eb4791bed41.patch?full_index=1"
+    sha256 "603f05c8c1eb3117f574eadbd6d0e9c52325245f5a4ba841ac7f0dfc13b37a34"
+  end
+
   def install
     # We build judy as static library, so we don't need to install it
     # into the real prefix
-    judyprefix = "#{buildpath}/resources/judy"
+    judyprefix = buildpath/"resources/judy"
 
     resource("judy").stage do
+      ENV.append_to_cflags "-fPIC" if OS.linux?
```

Comment:
This formula (`traildb`) doesn't ship shared libraries, right? If it does I'd be a little wary of doing this, and would consider linking `judy` dynamically instead of statically.

Reply:
In this case, you probably want to create a separate formula to reuse with `netdata`.

However, `judy` doesn't seem maintained anymore as last update was 2009, so that would mean introducing a potentially deprecated formula.

It is still available in major distros like Debian/Fedora/Arch (https://repology.org/project/judy/versions)